packages/shopify-codemod/test/fixtures/split-return-assignments/this.output.js

```diff
@@ -0,0 +1,5 @@
+foo(() => {
+  console.log(bar);
+  this.sho = this.doo = this.zoo;
```

Comment:
Why is this one moving the full assignment up, where the other ones that have more than two assignments break it down?